packages/plugins/experimental/plugin-wnfs/src/meta.tsx

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { FileCloud, type IconProps } from '@phosphor-icons/react';
+import React from 'react';
+
+import { pluginMeta } from '@dxos/app-framework';
+
+export const WNFS_PLUGIN = 'dxos.org/plugin/WNFS';
+
+export default pluginMeta({
+  id: WNFS_PLUGIN,
+  name: 'WNFS',
+  description: 'Upload & view files with WNFS.',
+  tags: ['experimental'],
+  iconComponent: (props: IconProps) => <FileCloud {...props} />,
```

Comment:
if you pull from `main` these were recently updated to be icon strings which index a sprite sheet. This will need to align with that for the icon to work now